Simple Calculator/Calculator.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows.Forms;
+
+namespace Simple_Calculator
+{
+    public partial class Calculator : Form
+    {
+        //variables which are being used
+        string valOne = "";
+        string valTwo = "";
+        string result = "";
+        string ArithOperator;
+        
+        public Calculator()
+        {
+            InitializeComponent();
+        }
+
+        private void button_Click(object sender, EventArgs e)
+        {
+            Button button = (Button)sender;
+            txtResult.Text = txtResult.Text + button.Text;
+        }
+
+        private void operators_Click(object sender, EventArgs e)
+        {
+            valOne = txtResult.Text;
+            Button button = (Button)sender;
+            ArithOperator = button.Text;
+            txtResult.Clear();
+        }
+
+        private void btnEqual_Click(object sender, EventArgs e)
+        {
+            valTwo = txtResult.Text;
+            txtResult.Clear();
+            CalCulateResult();
```

Comment:
get the return value of `CalCulateResult` and update the `txtResult` field.

Reply:
> get the return value of `CalCulateResult` and update the `txtResult` field.
like this?
`private void btnEqual_Click(object sender, EventArgs e)
        {
            valTwo = txtResult.Text;
            CalCulateResult();
            txtResult.Text = result;
        }`